ls1intum/Artemis
# `Exam Mode`: Make storing of any data for exam sessions configurable for administrators

### Checklist
#### General



### Motivation and Context / Description


Since our department for privacy/law said we can only store information for the feature described in here: https://github.com/ls1intum/Artemis/issues/5058#issuecomment-1133114391
I've created a property to disable any storing of data to the exam session: `info.student-exam-store-session-data`.

**Note: The new property replaces "info.browser-fingerprints-enabled"**

### Steps for Testing

Prerequisites:
- 1 Admin

1. Set the flag to false
2. Create an exam and participate as student
3. Verify in the DB that no data (IP, UserAgent, Fingerprint, Client Instance) has been stored (all shall be NULL)


### Review Progress





#### Code Review
#### Manual Tests
